mithril.js

```diff
@@ -1706,16 +1706,16 @@
 		var currentTarget = e.currentTarget || e.srcElement
 		var args
 
+		while (currentTarget && !/^a$/i.test(currentTarget.nodeName)) {
```

Comment:
My main concern is not about perf on this issue, it's accuracy. And i'm not sure of what it should do ;)
I guess is to route on first parent with an href (either &lt;a&gt; or &lt;area&gt; which are AFAIK the only 2 tags that act as links)